### PR TITLE
[EDGORDERS-81] - Release v3.0.0

### PR DESCRIPTION
**Pull request by imerabishvili**
## Purpose
https://folio-org.atlassian.net/browse/EDGORDERS-81

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
-## 2.10.0 - Unreleased
+## 3.1.0 - Unreleased
+
+## 3.0.0 - Released (Quesnelia R1 2024)
+The primary focus of this release was to update vertx version
+
+[Full Changelog](https://github.com/folio-org/edge-orders/compare/v2.9.0...v3.0.0)
+
+### Dependencies
+* Bump `vertx` from `4.3.4` to `4.5.4`
```
Comment (SerhiiNosko) [inline NEWS.md:9]: lets add here also new version of edge-common and update description mentioning edge-common

Comment (imerabishvili) [reply NEWS.md:9]: added

**Review by SerhiiNosko — APPROVED** [review]

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
-## 2.10.0 - Unreleased
+## 3.1.0 - Unreleased
+
+## 3.0.0 - Released (Quesnelia R1 2024)
+The primary focus of this release was to update vertx and edge-common versions
+
+[Full Changelog](https://github.com/folio-org/edge-orders/compare/v2.9.0...v3.0.0)
```
Comment (azizbekxm) [inline NEWS.md:6]: there is stories to add
https://folio-org.atlassian.net/projects/EDGORDERS/versions/11580/tab/release-report-all-issues

Comment (azizbekxm) [reply NEWS.md:6]: and also fix versions should be changed of stories

Comment (SerhiiNosko) [reply NEWS.md:6]: these 2 stories are about the same, as for me it's ok to include only single of them

Comment (imerabishvili) [reply NEWS.md:6]: These stories are module version updates so according to guidelines they are included in `Dependencies` section but not in `Stories` section as there are no code changes.

Comment (azizbekxm) [reply NEWS.md:6]: then, let's include at least one of them in stories section to show some effort made in this module

**Review by azizbekxm — APPROVED** [review]

**Pull request merged**